content/docs/components/datepicker.mdx

```diff
@@ -69,6 +69,12 @@ Use the `inline` prop to show the datepicker component without having to click i
 
 <Example name="datepicker.inline" />
 
+## Date value
+
+Use the `value` prop to provide a custom value to datepickers date. Providing null will clear the selected date.
```

Comment:
Something like

```
Use `<Datepicker value={}` to create a controlled `<Datepicker />`. Pass `null` to reset input.
```

Reply:
Made a small change to your suggestion, guess fits better.